RNTester/js/components/Background.js

```diff
@@ -0,0 +1,31 @@
+/**
```

Comment:
Ideally , this file should render an SVG or render 3 native views.

- Did not use an SVG as it will cause issues will the migration of the app.
- Did not render the 3 views as I was unsure if this would be the right way to implement the style.

- Currently, using a @4x PNG file.